assets/css/nav/_top_nav_mobile.scss

```diff
@@ -0,0 +1,108 @@
+.m-top-nav-mobile-content {
+  display: flex;
+  box-sizing: border-box;
+  padding: 0.75rem 1.5rem 0.5rem 1.5rem;
+  justify-content: space-between;
+  background-color: $color-gray-50;
+  border-bottom: 1px solid $color-gray-300;
+  z-index: 108;
+}
+
+.m-top-nav-mobile-overlay {
+  position: fixed;
+  padding: 0px;
+  margin: 0px;
+  display: none;
```

Comment:
For what it's worth, historically for rendering things that should only display in some situations we have used conditionals in the JSX rather than the method of setting / unsetting `display: none` via CSS. I think this approach is fine too, and I'd actually be curious if there's a strong reason to prefer one over the other going forward.